storage/s3_sdk/list_gcs_buckets_test.go

```diff
@@ -17,29 +17,78 @@ package s3sdk
 
 import (
 	"bytes"
+	"context"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"os"
 	"strings"
 	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
+var serviceAccountEmail = os.Getenv("GOLANG_SAMPLES_SERVICE_ACCOUNT_EMAIL")
+var storageClient *storage.Client
```

Comment:
In general, try to avoid global variables in Go. This isn't a bad case since it's a test. But, it would be better to get the email in where it's needed and pass the client as an argument where it's needed. It makes the dependencies and control flow very clear.